### PR TITLE
feat(backend): deployment for backend

### PR DESCRIPTION
**Pull request by ulfgebhardt**
## 🍰 Pullrequest

deployment for backend

### Issues

- None

### Todo

- [X] None

### CODE REVIEW

File: backend/package-lock.json
Comment (mahula) [inline backend/package-lock.json:1]: Why did the package lock file change?
Where are the package.json file changes related to that?

Comment (ulfgebhardt) [reply backend/package-lock.json:1]: Someone did not properly check in the lockfile

File: deployment/migrate.database.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# Find current directory & configure paths
+SCRIPT_PATH=$(realpath $0)
+SCRIPT_DIR=$(dirname $SCRIPT_PATH)
+PROJECT_ROOT=$SCRIPT_DIR/../backend
+
+# Build the project
+cd $PROJECT_ROOT
+npm run db:migrate
```
Comment (Mogge) [inline deployment/migrate.database.sh:10]: May be we need to create a new npm command:
https://www.prisma.io/docs/orm/reference/prisma-cli-reference#migrate-deploy

**Review by Mogge — CHANGES_REQUESTED** [review]
Please check mu suggestion

File: deployment/migrate.database.sh (hunk 1 above)
Comment (ulfgebhardt) [reply deployment/migrate.database.sh]: Fixed, tested & works

**Review by Mogge — APPROVED** [review]

**Pull request merged**